en_ID/pages/progress.md

```diff
@@ -47,6 +52,14 @@ const showData = () => {
     `
   })
 }
+function showProgress() {
```

Comment:
since most of our code is using es6, you can use arrow-function style instead of plain old function.

So instead of `function showProgress()` it can be `showProgress = () => {}`
So instead of `function hideProgress()` it can be `hideProgress = () => {}`